compose/utils.py

```diff
@@ -49,6 +49,9 @@ def split_buffer(stream, splitter=None, decoder=lambda a: a):
     splitter = splitter or line_splitter
     buffered = six.text_type('')
 
+    if type(stream) is bytes:
+        stream = stream.decode("utf-8")
```

Comment:
I see what's happening. I missed it before but setting `stream=False` is actually wrong.

I think we'll want to remove this, and I'll add another comment for the other fix.